mypy_primer/projects.py

```diff
@@ -1510,6 +1510,13 @@ def get_projects() -> list[Project]:
             deps=["packaging"],
             expected_success=("pyright",),
         ),
+        Project(
+            location="https://github.com/strawberry-graphql/strawberry",
+            mypy_cmd="{mypy} {paths} --config-file=",
+            pyright_cmd="{pyright} {paths}",
+            paths=["strawberry"],
+            deps=["graphql-core", "typing-extensions", "python-dateutil", "packaging"],
```

Comment:
It probably doesn't matter much since it's a light dependency that's quick to install, but I doubt we need to explicitly include `typing-extensions` as a dependency here. The stubs for `typing-extensions` are provided by typeshed (it pretends `typing-extensions` is part of the standard library so that we can e.g. use `typing-extensions` backports in `builtins.pyi` etc.)

```suggestion
            deps=["graphql-core", "python-dateutil", "packaging"],
```

Reply:
It's listed as a dependency for a few other projects, so I wasn't sure...